libbeat/dashboards/dashboards/importer.go

```diff
@@ -0,0 +1,535 @@
+package dashboards
```

Comment:
First `dashboards` is the current `import_dashboards` main package. The second is where I gathered most of the code. After we remove `import_dashboards` we can move the package one level up.